src/test/scala-2.11/markets/participants/TestOrderIssuer.scala

```diff
@@ -28,32 +28,32 @@ import scala.concurrent.duration.FiniteDuration
 import scala.concurrent.ExecutionContext.Implicits.global
 
 
-case class TestLiquidityDemander(initialDelay: FiniteDuration,
-                                 interval: Option[FiniteDuration],
-                                 markets: mutable.Map[Tradable, ActorRef],
-                                 tickers: mutable.Map[Tradable, Agent[Tick]])
-  extends LiquidityDemander[MarketOrderTradingStrategy] {
+case class TestOrderIssuer(initialDelay: FiniteDuration,
+                           interval: Option[FiniteDuration],
+                           markets: mutable.Map[Tradable, ActorRef],
+                           tickers: mutable.Map[Tradable, Agent[Tick]])
+  extends OrderIssuer {
 
   interval match {
     case Some(duration) =>
-      context.system.scheduler.schedule(initialDelay, duration, self, SubmitMarketBidOrder)
+      context.system.scheduler.schedule(initialDelay, duration, self, SubmitAskOrder)
     case None =>
-      context.system.scheduler.scheduleOnce(initialDelay, self, SubmitMarketBidOrder)
+      context.system.scheduler.scheduleOnce(initialDelay, self, SubmitAskOrder)
   }
 
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [It's preferred to use immutable collections instead of mutable ones](https://www.codacy.com/app/davidrpugh/markets-sandbox/file/2363134682/issues/source?bid=2921361&fileBranchId=3219434#l44)